src/System.Net.Http.WinHttpHandler/src/System.Net.Http.WinHttpHandler.msbuild

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You should follow the pattern used by the other projects here.  Where we have files that are only used for one platform, in the main project we put an item group headed with a condition for Targets<OSGroup> and then the compile includes under that.  The .msbuild file is confusing